### PR TITLE
Support Scala-2.13.0-RC3

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

File: project/plugin-scalajs.sbt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.27")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.28")
```
Comment (rossabaker) [inline project/plugin-scalajs.sbt]: 0.6.27 is not published for RC3, whereas 0.6.28 is published for all the supported Scala versions.

Comment (sarahgerweck) [reply project/plugin-scalajs.sbt]: 👍 unless somebody wants to argue they need new builds against old versions of scalajs, I'm happy to stay up to date.

**Comment by rossabaker** [comment]
The [failing builds](https://travis-ci.org/Log4s/log4s/builds/540954560) are all the old Multiple Test Appenders issue.

**Comment by sarahgerweck** [comment]
Version 1.8.1 with this change went out to Sonatype. It should show up in Maven Central within half an hour. Thanks again @rossabaker. I'll have to take a look at the build issue. I think I have a solution in mind that may work.

**Pull request merged**